### DISCUSSION TITLE
Current Priorities - Top 5

### DISCUSSION THREAD

**Discussion by ooliviarr** [govhackaustralia/hackerspace3#236] [General]
Our top 5 priorities will be:

a) tech debt and ensuring all existing infrastructure will stand the test of time
b) new website
c) data APIs (data team to lead)
d) LinkedIn badges
e) more Slack integrations.

No other major new features in scope. Any new scope will need to be prioritised against the current top 5.

**Comment by cassar** [comment]
Just seeing this now @ooliviarr 😅

To clarify, point "a) tech debt" we're talking about the the UI/accessibility upgrades?
